packages/components/data-table-manager/src/data-table-manager.stories.tsx

```diff
@@ -255,25 +667,65 @@ export const BasicExample: Story = (args) => {
     ...columnManagerButtons,
   };
 
+  // const NestedComponent = (props: { id: string }) => {
```

Comment:
These were deliberately commented out for the second phase of the work that involves the nested rows